terraform/environments/electronic-monitoring-data/lambdas/output_file_structure_as_json_from_zip.py

```diff
@@ -0,0 +1,104 @@
+import base64
+import datetime
+import gzip
+import io
+import json
+from logging import getLogger
+
+from aws_lambda_powertools.utilities.streaming.transformations import ZipTransform
+from aws_lambda_powertools.utilities.streaming.s3_object import S3Object
+import boto3
+
+
+logger = getLogger(__name__)
+
```

Comment:
```suggestion
logger.setLevel(logging.INFO)
```
makes reading cloudwatch logs easier

Reply:
I'll have a look up what that does, but will commit for now.